Terminal-iOS/Set/Set/SetRemoteManager.swift

```diff
@@ -63,4 +63,37 @@ class SetRemoteManager: SetRemoteDataManagerInputProtocol {
                 }
             }
     }
+    func postLogout(userID: String) {
+        TerminalNetworkManager
+            .shared
+            .session
+            .request(TerminalRouter.logout(userID: userID))
+            .validate()
+            .responseJSON { response in
+                switch response.result {
+                case .success(let value):
+                    let json = JSON(value)
+                    let data = "\(json)".data(using: .utf8)
+                    do {
+                        let result = try JSONDecoder().decode(BaseResponse<String>.self, from: data!)
+                        if result.message != nil {
+                            self.interactor?.postLogoutResult(result: result)
+                        }
+                    } catch {
+                        
+                    }
```

Comment:
```suggestion
            .responseData { response in
                switch response.result {
                case .success(let data):
                    do {
                        let result = try JSONDecoder().decode(BaseResponse<String>.self, from: data)
                        if result.message != nil {
                            self.interactor?.postLogoutResult(result: result)
                        }
                    } catch {
                        
                    }
```
이런식으로 변경이 가능해서 코드 라인이 더 줄어들어서 좋은듯 싶어요.

이렇게 하는게 어떤지요??

Reply:
음 다른부분들 다 바꾸나용 그럼??

Reply:
지금 구조가 responseJSON을 통해서  json -> String -> data 이구조로 만들고 사용하고 있는데

responseData 를 통하면 바로 data 를 할당 할 수 있어서요 작업하다가 보이는 부분마다 고치는게 어떤지??

Reply:
그렇게 되면 SwiftyJSON 라이브러리도 import 하지 않아도 되어서 좋은 것 같구요.

Reply:
decode에 from에들어갈 NSData형으로 Any로 떨어지는 NSDictionary를 캐스팅 해줘야할것 같아요

Reply:
다음 작업 때 추가하겠습니다 일단 머지